common/lib/xblock/xblock_discussion/xblock_discussion.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+"""
+Discussion XBlock
+"""
+import json
+import logging
+
+from xblockutils.resources import ResourceLoader
+from xblockutils.studio_editable import StudioEditableXBlockMixin
+
+from xblock.core import XBlock
+from xblock.fields import Scope, String, UNIQUE_ID
+from xblock.fragment import Fragment
+
+log = logging.getLogger(__name__)
+loader = ResourceLoader(__name__)  # pylint: disable=invalid-name
+
+
+@XBlock.needs('user')
+class DiscussionXBlock(XBlock, StudioEditableXBlockMixin):
+    """ Provides functionality similar to discussion XModule in inline mode """
+
+    discussion_id = String(scope=Scope.settings, default=UNIQUE_ID, force_export=True)
+    display_name = String(
+        display_name="Display Name",
```

Comment:
In `discussion_module.py`, these strings were all marked for i18n scraping but now they're not - that is a regression.